meta-balena-common/recipes-bsp/u-boot/patches/env_resin.h

```diff
@@ -149,7 +149,7 @@
               "fi;\0" \
        \
        "set_os_cmdline=" \
-               "setenv os_cmdline ${base_os_cmdline} ${bootparam_flasher};\0" \
+               "setenv os_cmdline ${base_os_cmdline} ${bootparam_flasher} ${resin_extra_cmdline_args};\0" \
```

Comment:
I don't think we should use the `resin` brand in any new variable. Maybe something generic in symmetry to `base_os_cmdline` like `extra_os_cmdline`?

Reply:
Thanks Alex, indeed, extra_os_cmdline sounds better